tests/test_image.py

```diff
@@ -0,0 +1,260 @@
+
+
+def assert_objects_at_positions(image, expected_positions, expected_brightness_values, pixel_radius=10, rtol=0.1):
+    """Sum the brightness values of squares of side length `2*pixel_radius` centered at `expected_positions` and compare against `expected_brightness_values`."""
+    brightness_values = np.empty_like(expected_brightness_values)
+    for i, (col, row) in enumerate(expected_positions):
+        neighbourhood = image[row-pixel_radius:row+pixel_radius, col-pixel_radius:col+pixel_radius]
+        brightness_values[i] = np.sum(neighbourhood)
+        print("Object: ", i, expected_brightness_values[i], brightness_values[i])
+    np.testing.assert_allclose(brightness_values, expected_brightness_values, rtol=rtol)
+
+
+def create_test_config(
+    image_type="LSST_PhotonPoolingImageBuilder",
+    exptime: float = 30.0,
+    enable_diffraction: bool = True,
+    band="r",
+    rottelpos=20.0 * galsim.degrees,
+    fft_sb_thresh: float=10000.0
+):
+    bandpass = galsim.Bandpass(f"LSST_{band}.dat", wave_type="nm").withZeropoint("AB")
+    opsim_data = imsim.OpsimDataLoader(str(INSTCAT))
+    boresight = galsim.CelestialCoord(
+        opsim_data['rightascension'] * galsim.degrees, opsim_data['declination'] * galsim.degrees
+    )
+    telescope = imsim.load_telescope(f"LSST_{band}.yaml", rotTelPos=rottelpos)
+    wcs_factory = imsim.BatoidWCSFactory(
+        boresight,
+        obstime=Time.strptime("2022-08-06 06:50:59.337600", "%Y-%m-%d %H:%M:%S.%f"),
+        telescope=telescope,
+        wavelength=622.3195217611445,  # nm
+        camera=imsim.get_camera(),
+        temperature=280.0,
+        pressure=72.7,
+        H2O_pressure=1.0,
+    )
+    det_name = "R22_S11"
+    camera = imsim.get_camera()[det_name]
+    wcs = wcs_factory.getWCS(det=camera)
+
+    alt_az = {
+        "altitude": 88.0 * galsim.degrees,
+        "azimuth": 73.7707957 * galsim.degrees,
+    }
+    if enable_diffraction:
+        optics_args = {
+            "type": "RubinDiffractionOptics",
+            "det_name": "R22_S11",
+            "disable_field_rotation": exptime == 0.0,
+            **alt_az,
+        }
+    else:
+        optics_args = {"type": "RubinOptics",
+                       "det_name": "R22_S11",
+                       }
+    config = {
+        "input": {
+            "telescope": {
+                "file_name": f"LSST_{band}.yaml",
+                "rotTelPos": rottelpos,
+            },
+            "instance_catalog": {
+                "file_name": str(INSTCAT),
+                "sed_dir": str(SED_DIR),
+            }
+        },
+        "det_name": det_name,
+        "output": {"camera": "LsstCam"},
+        "_icrf_to_field": wcs_factory.get_icrf_to_field(camera),
+        "gal": {
+            "type": "DeltaFunctions",
+            # 10 FFT objects, 10 photon shooting objects (including 1 faint object)
+            "fluxes": [5.0e4] * 10 + [5.0e3] * 9 + [1.0],
+        },
```

Comment:
This would be simpler as
```
"gal": {
    "type": "DeltaFunction",
    "sed": {
            "file_name": "vega.txt",
            "wave_type": "nm",
            "flux_type": "flambda",
            "norm_wavelength": 500,
            "norm_flux_density": [5.0e4] * 10 + [5.0e3] * 9 + [1.0],
        }
    }
```
No need for a custom type.

Reply:
Thanks, that's much simpler! Done.